assets/src/application/ui/calendars/dateDisplay/CalendarDateSwitcher/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import classNames from 'classnames';
+import { parseISO } from 'date-fns';
+import { __ } from '@wordpress/i18n';
+
+import { DisplayStartOrEndDate } from '@sharedServices/filterState';
+import { PLUS_ONE_MONTH, PLUS_TWO_MONTHS } from '@sharedConstants/defaultDates';
+import { BiggieCalendarDate, CalendarDateRange } from '../';
+
+interface Props {
+	className?: string;
+	displayDate: DisplayStartOrEndDate;
+	endDate: string;
+	footerText?: string;
+	headerPrefix?: string;
+	headerText?: string;
+	showDate?: boolean;
+	startDate: string;
+}
+
+const CalendarDateSwitcher: React.FC<Props> = React.memo(
+	({
+		className,
+		displayDate = DisplayStartOrEndDate.start,
+		footerText,
+		headerPrefix,
+		headerText,
```

Comment:
let's just use the `headerText` prop only 

then later on we'll replace the text being fed into after working on https://github.com/eventespresso/event-espresso-core/projects/6#card-35163011